templates/geosafe/analysis/create.html

```diff
@@ -133,33 +133,33 @@
         }
 
         .option .set .badge {
-            background-color: #50cf5c;
+            background-color: #1a6c38;
         }
 
         .option .has-children:not(.set) .badge {
-            background-color: #3db2dd;
+            background-color: #2aa858;
```

Comment:
if we decided to set the color for not selected layer to green, we need to change the color for selected layer (it's also green) as it's not obvious for the user which one means what IMO. cc @carina1000